project/externals/cyrus-sasl.cmake

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
```

Comment:
I noticed that the Copyright of this repo is not up-to-date. Does it need to be consistent with the repo of `nebula`?

Reply:
Nice catch. It should be 2021 here

Reply:
Nice catch. It should be 2021 here

Reply:
Also need to remove the CC 1.0 part

Reply:
> Also need to remove the CC 1.0 part
ok，I'll deal with it later. ☺️